linux/scripts/dist.sh

```diff
@@ -53,36 +59,29 @@ mkdir -p dist
 for proj in ${autotool_projects}; do
     echo "configure $proj"
     cd $proj
-    oldvers=`cat VERSION`
-    vers=`./configure -version|grep -E 'kmfl|keyman'|grep -Po "${proj} configure \K[^a-z]*"`
-    echo "$vers" > VERSION
     rm -rf ../build-$proj
     mkdir -p ../build-$proj
     cd ../build-$proj
     GTK_CFLAGS="-I/usr/include/gtk-3.0" GTK_LIBS="-lgtk-3 -lgdk-3" JSON_GLIB_CFLAGS="-I/usr/include/json-glib-1.0" JSON_GLIB_LIBS="-ljson-glib-1.0" KEYMAN_PROC_CFLAGS="-I/usr/include/keyman" KEYMAN_PROC_LIBS="-lkmnkbp" ../$proj/configure
     make dist
     mv *.tar.gz ../dist
     cd $BASEDIR
-    echo "$oldvers" > VERSION
 done
 
 for proj in ${extra_projects}; do
     # dist for keyman-config
     if [ "${proj}" == "keyman-config" ]; then
-
         cd keyman-config
-        vers=`cat ../../VERSION.md`
         echo "3.0 (native)" > debian/source/format
-        dch keyman-config --newversion ${vers} --force-bad-version --nomultimaint
+        dch keyman-config --newversion ${VERSION} --force-bad-version --nomultimaint
         dpkg-source --tar-ignore=*~ --tar-ignore=.git --tar-ignore=.gitattributes \
             --tar-ignore=.gitignore --tar-ignore=experiments --tar-ignore=debian \
             --tar-ignore=__pycache__ -Zgzip -b .
-        mv ../keyman-config_*.tar.gz ../dist/keyman-config.tar.gz
+        mv ../keyman-config_*.tar.gz ../dist/keyman-config-${VERSION}.tar.gz
```

Comment:
Before this PR gets merged, we'll need to update the Keyman - Linux Release-14.0 config

There's a TODO on step 6 (l.23)